packages/autorest.python/autorest/codegen/models/response.py

```diff
@@ -59,6 +59,13 @@ def __init__(
         self.type = type
         self.nullable = yaml_data.get("nullable")
 
+    @property
+    def result_property(self) -> str:
```

Comment:
we should have the full result type in the emitter

Reply:
It is just special logic for [Action LRO[(https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#long-running-action-operations). Otherwise, we need to handle it in azure-core. In view of that typespec is not GAed, I think it is better to do like here and move it to azure-core if needed in the future.

Reply:
@msyyc mark has agreed this is a bug and he's looking into it. I think for now we can leave lro-rpc as unimplemented and just commit the lro logic. no special case for lro rpc, just always set the final response type of the LROPoller to be `logicalResult` in the emitter